src/components/HomeViewUsers/index.css

```diff
@@ -0,0 +1,38 @@
+.img__user {
+  height: 100px;
+  width: 100px;
+  border-radius: 50%;
+  border: 2px solid #04a9e0;
+}
+
+h2 {
+  font-family: "Arial", "Open Sans";
+  color: #828282;
+  text-transform: capitalize;
+}
+
+.view__users {
+  display: grid;
+  width: 400px;
+  height: 120px;
+  border: solid 1px rgb(241, 241, 241);
+  /* border: solid 1px rgb(63, 58, 58); */
+  background-color: #f4f4f4;
+  border-radius: 15% / 50%;
+  box-shadow: 10px 5px 5px rgba(0, 0, 0, 0.1);
+}
+
+ul {
+  display: grid;
+  grid-row-gap: 40px;
+  grid-template-columns: 20px;
+  padding: 20px;
+}
+
+.user__list {
```

Comment:
this is not correct naming according to BEM.
We should start wit the component name.